scipy/signal/tests/test_windows.py

```diff
@@ -213,5 +213,15 @@ def test_windowfunc_basics():
         window(6, *params, sym=False)
 
 
+def test_kaiser_derived():
+    M = 100
+    w = signal.kaiser_derived(M)
+
+    # Test for Princen-Bradley condition
+    assert_allclose(w[:M//2]**2 + w[-M//2:]**2, 1.)
+
+    # Assert ValueError for odd window length
+    assert_raises(ValueError, signal.kaiser_derived, M + 1)
+
```

Comment:
Style nit: There should be one more blank line between here (between the end of the function definition and the `if` statement`).